### PR TITLE
document command necessary before running `jhipster import-jdl`

### PR DESCRIPTION
**Pull request by SoniaLahi**
fix https://github.com/jhipster/generator-jhipster/issues/12553

Adding documentation on command to run before generation of entities

---

Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by pascalgrimaud — APPROVED** [review]

**Comment by mshima** [comment]
@pascalgrimaud IMO we should document `--skip-jhipster-dependencies` instead.

**Comment by pascalgrimaud** [comment]
I'm discovering new things :)
What is `--skip-jhipster-dependencies` plz ?

**Comment by mshima** [comment]
> I'm discovering new things :)
> What is `--skip-jhipster-dependencies` plz ?

https://github.com/jhipster/generator-jhipster/pull/12324
When running a linked generator-jhipster, use it and it will prevent to install locally.

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]
@SoniaLahi : can you do the change request from @mshima plz ? It's better to use `--skip-jhipster-dependencies` instead of `npm link generator-jhipster`

**Comment by mshima** [comment]
Maybe you can find blueprint doc for jhipster 7 useful: https://github.com/jhipster/jhipster.github.io/pull/998/files

**Comment by SoniaLahi** [comment]
I will do it :)

**Comment by pascalgrimaud** [comment]
any progress on this @SoniaLahi ?

**Comment by SoniaLahi** [comment]
Finishing it today !

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -228,11 +228,13 @@ This will do a symbolic link from the global `node_modules` version to point to
 For testing, you will want to generate an application, and there is a specific issue here: for each application, JHipster installs a local version of itself. This is made to enable several applications to each use a specific JHipster version (application A uses JHipster 3.1.0, and application B uses JHipster 3.2.0).
 
 To overcome this you need to run `npm link generator-jhipster` on the generated project folder as well, so that the local version has a symbolic link to the development version of JHipster.
+Also add the option `--skip-jhipster-dependencies` to generate the application ignoring the JHipster dependencies (otherwise a released version with be installed each time npm install/ci is called). You can later on re-add the dependency with the command `jhipster --no-skip-jhipster-dependencies`.
```
Comment (pascalgrimaud) [inline CONTRIBUTING.md:231]: with be -> will be

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (SoniaLahi) [reply CONTRIBUTING.md:231]: the error is also here:  https://github.com/jhipster/jhipster.github.io/pull/998/files#diff-582158a3a484757228416d0a6de1af7eR223

Comment (SoniaLahi) [reply CONTRIBUTING.md]: I'm fixing it too

Comment (SoniaLahi) [reply CONTRIBUTING.md]: PR : https://github.com/jhipster/jhipster.github.io/pull/1018

Comment (pascalgrimaud) [reply CONTRIBUTING.md]: thanks :)

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**